requirements_all.txt

```diff
@@ -1768,7 +1768,7 @@ python-gitlab==1.6.0
 python-hpilo==4.3
 
 # homeassistant.components.izone
-python-izone==1.1.3
+python-izone==1.1.4
```

Comment:
Thanks for bumping the dependency! However, could you please update the PR description to contain at least one (or multiple) of the following:

- A link to the release notes of this package version, and all versions in between.
- A link to the changelog of this package.
- A link to a Git(Hub) diff/compare view from the current version to the bumped version.

It seems like the PR description contains none of these.

Reply:
PR updated with all of the above.